aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -462,37 +503,53 @@ module aptos_framework::account {
         if (account_scheme == ED25519_SCHEME) {
-            let pubkey = ed25519::new_unvalidated_public_key_from_bytes(account_public_key_bytes);
-            let expected_auth_key = ed25519::unvalidated_public_key_to_authentication_key(&pubkey);
+            let pubkey =
+                ed25519::new_unvalidated_public_key_from_bytes(account_public_key_bytes);
+            let expected_auth_key = ed25519::unvalidated_public_key_to_authentication_key(
+                &pubkey
+            );
             assert!(
                 account_resource.authentication_key == expected_auth_key,
-                error::invalid_argument(EWRONG_CURRENT_PUBLIC_KEY)
+                error::invalid_argument(EWRONG_CURRENT_PUBLIC_KEY),
             );
 
-            let rotation_capability_sig = ed25519::new_signature_from_bytes(rotation_capability_sig_bytes);
+            let rotation_capability_sig = ed25519::new_signature_from_bytes(
+                rotation_capability_sig_bytes
+            );
             assert!(
-                ed25519::signature_verify_strict_t(&rotation_capability_sig, &pubkey, proof_challenge),
-                error::invalid_argument(EINVALID_PROOF_OF_KNOWLEDGE)
+                ed25519::signature_verify_strict_t(
+                    &rotation_capability_sig, &pubkey, proof_challenge
+                ),
+                error::invalid_argument(EINVALID_PROOF_OF_KNOWLEDGE),
             );
         } else if (account_scheme == MULTI_ED25519_SCHEME) {
-            let pubkey = multi_ed25519::new_unvalidated_public_key_from_bytes(account_public_key_bytes);
-            let expected_auth_key = multi_ed25519::unvalidated_public_key_to_authentication_key(&pubkey);
+            let pubkey =
+                multi_ed25519::new_unvalidated_public_key_from_bytes(
+                    account_public_key_bytes
+                );
+            let expected_auth_key =
+                multi_ed25519::unvalidated_public_key_to_authentication_key(&pubkey);
             assert!(
                 account_resource.authentication_key == expected_auth_key,
-                error::invalid_argument(EWRONG_CURRENT_PUBLIC_KEY)
+                error::invalid_argument(EWRONG_CURRENT_PUBLIC_KEY),
             );
 
-            let rotation_capability_sig = multi_ed25519::new_signature_from_bytes(rotation_capability_sig_bytes);
+            let rotation_capability_sig =
+                multi_ed25519::new_signature_from_bytes(rotation_capability_sig_bytes);
             assert!(
-                multi_ed25519::signature_verify_strict_t(&rotation_capability_sig, &pubkey, proof_challenge),
-                error::invalid_argument(EINVALID_PROOF_OF_KNOWLEDGE)
+                multi_ed25519::signature_verify_strict_t(
+                    &rotation_capability_sig, &pubkey, proof_challenge
+                ),
+                error::invalid_argument(EINVALID_PROOF_OF_KNOWLEDGE),
```

Comment:
Prefer these to be on different lines
```move
                multi_ed25519::signature_verify_strict_t(
                    &rotation_capability_sig,
                    &pubkey,
                    proof_challenge
                ),
```

Reply:
If that's the case, I'm quite in agreement, although the downside is that it will increase the number of lines. In our next version, we will take more aggressive action with function calls, adopting a multi-line mode for any calls with more than two parameters, with one parameter per line.